stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: lts-6.14
+resolver: lts-6.23
```

Comment:
This bump is needed to get a newer version of conduit-combinators for tar-conduit. If this is a problem, two alternatives:

* Switch back to the tar package
* Modify tar-conduit to have a more lenient lower bound